plugins/modules/rabbitmq_user_limits.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2013, Chatham Financial <oss@chathamfinancial.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: rabbitmq_user_limits
+short_description: Manage RabbitMQ user limits
+description:
+  - Manage the state of user limits in RabbitMQ.
```

Comment:
```suggestion
  - Manage the state of user limits in RabbitMQ. Supported since RabbitMQ version 3.8.10.
```
How about this or something with the same meaning?